charts/sn-platform-slim/templates/zookeeper/_zookeeper.tpl

```diff
@@ -22,6 +22,23 @@ Define the pulsar zookeeper
 {{- end -}}
 {{- end -}}
 
+{{/*
+Define the pulsar configurationStore
+*/}}
+{{- define "pulsar.configurationStore.connect" -}}
+{{$configurationStore:=.Values.pulsar_metadata.configurationStoreServers}}
+{{- if and (not .Values.components.zookeeper) $configurationStore }}
+{{- $configurationStore -}}
+{{ else }}
+{{- if not (and .Values.tls.enabled .Values.tls.zookeeper.enabled) -}}
```

Comment:
Seems like it can use `if else` here.
```yaml
{{- if and .Values.tls.enabled .Values.tls.zookeeper.enabled -}} 
{{ template "pulsar.zookeeper.service" . }}:{{ .Values.zookeeper.ports.client }}
{{ else }}
{{ template "pulsar.zookeeper.service" . }}:{{ .Values.zookeeper.ports.clientTls }}
{{ end }} 
```

Reply:
Updated,PTAL